compiler/driver.ml

```diff
@@ -970,9 +990,10 @@ module Commands = struct
   let c options includes output optimize check_invariants =
     let prg, type_ordering, _ren_ctx =
       Passes.scalc options ~includes ~optimize ~check_invariants
-        ~closure_conversion:true ~keep_special_ops:true
+        ~closure_conversion:true ~keep_special_ops:false
```

Comment:
Well, actually, for the time being I've been reusing it with a slightly different meaning... basically, it affects closure conversion with the idea is that when turned on, the functional arguments to operators (first parameter of `Fold`, `Map`, etc.) should not be converted because they will be treated specifically by the backend (typically, turned to loops).